spec/src/indexSpec.js

```diff
@@ -137,7 +137,21 @@ describe('cloudwatch-metrics', function() {
       metric.put(2, 'metricName', [{Name:'ExtraDimension',Value: 'Value'}]);
     });
   });
-
+  describe('unit checking', function() {
+    it('should error for unknown units', function () {
+      var error = false;
+      try {
+        var metric = new cloudwatchMetric.Metric('namespace', 'UnknownUnit', [{
+          Name: 'environment',
+          Value: 'PROD'
+        }]);
+        metric.put(1, 'metricName', [{Name:'ExtraDimension',Value: 'Value'}]);
+      }catch(e){
+        error = true;
```

Comment:
Please check the `Error` has an error message that makes sense (e.g. test it against `/unit/i`)